{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -50,6 +50,7 @@ repos:
     rev: 6.1.0
     hooks:
       - id: flake8
+        args: [ '--config=setup.cfg' ]
```

Comment:
Do we need that? I was under the impression that [it's a location](https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations) where flake8 looks out of the box

Reply:
Thank you for leaving a comment. I appreciate it. I have confirmed that this is the location that flake8 checks by default.